registration/views.py

```diff
@@ -93,10 +208,7 @@ def post(self, request):
         form = self.form(request.POST)
 
         if not form.is_valid():
-            error_list = functools.reduce(sum, list(form.errors.values()))
-            logging.error('Received invalid RegisterForm')
-            messages.error(request, "\n\n".join(error_list))
-            return render(request, self.template_name, {"form": form})
+            return _validation_errors(request, self.template_name, form)
```

Comment:
Are you testing the errors reported in case of an invalid form? It seems to be covered by the tests but I cannot find the test or (re-)produce an error/invalid form.

Reply:
[Test 3](https://github.com/gocept/ok_tools/pull/23/files#diff-4a1851aeebbd7657cedbfe83449a69360998cc219063ec3681880b7937ac288cR307) for the EditProfileView tests the form with an invalid email address.